### PR TITLE
add support for swarm config (possibly stun/turn servers)

### PR DESCRIPTION
**Pull request by sujeebant**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -14,6 +14,17 @@ const debug = (function () {
   }
 })()
 
+const createSwarm = (hub) => {
+  const swOpts = {}
+  if (!swarm.WEBRTC_SUPPORT) {
+    swOpts.wrtc = require('wrtc')
+  }
+
+  swOpts.config = config.SWARM_CONFIG
```
Comment (G-Ray) [inline src/index.js]: if wrtc is set, it will be overridden here

File: src/index.js [hunk 2]
```diff
@@ -14,6 +14,17 @@ const debug = (function () {
   }
 })()
 
+const createSwarm = (hub) => {
```
Comment (G-Ray) [inline src/index.js]: What about: 
```js
const createSwarm = (hub) =>
  swarm(hub, {
    wrtc: !swarm.WEBRTC_SUPPORT ? require('wrtc') : null
    config: config.SWARM_CONFIG
  })
```

Comment (sujeebant) [reply src/index.js]: Initializing the swOpts and filling each attribute seemed more explicit, what do you think ?

Comment (G-Ray) [reply src/index.js]: nope I find it's more functional to declare and initialize once. It's less error prone

Comment (G-Ray) [reply src/index.js]: By defining the object at the same time we declare it, it's also easier to know exactly what's inside the object

**Pull request merged**